postman/uady-dwaas-back-local.postman_environment.json

```
@@ -1,15 +1,21 @@
 {
-	"id": "4b240ada-de86-426a-88c6-ffaeccf5af02",
+	"id": "72b8d9c5-fd18-4c8e-8771-2b436f8bde1c",
 	"name": "uady-dwaas-back-local",
 	"values": [
 		{
 			"key": "baseUrl",
 			"value": "http://localhost:8080/api",
 			"type": "default",
 			"enabled": true
+		},
+		{
+			"key": "Token",
```

Para el nombre de las variables ando usando camelCase, así que un nombre que creo que es más consistente y descriptivo es:

```suggestion
			"key": "jwt",
```